lib/kasket/configuration_mixin.rb

```diff
@@ -79,7 +79,16 @@ def has_kasket_on(*args)
       extend ReadMixin unless methods.map(&:to_sym).include?(:find_by_sql_with_kasket)
     end
 
+    def kasket_expires_in(time)
```

Comment:
we tried that first, but it looks unsymetric:

has_kasket
self.kasket_expires_in = 5.minutes

vs

has_kasket
kaket_expires_in 5.minutes

On Tue, Sep 1, 2015 at 3:45 PM, Mick Staugaard <notifications@github.com>
wrote:

> In lib/kasket/configuration_mixin.rb
> <https://github.com/zendesk/kasket/pull/19#discussion_r38481497>:
>
> > @@ -79,7 +79,16 @@ def has_kasket_on(*args)
> >        extend ReadMixin unless methods.map(&:to_sym).include?(:find_by_sql_with_kasket)
> >      end
> >
> > +    def kasket_expires_in(time)
>
> this should be called kasket_expires_in=
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zendesk/kasket/pull/19/files#r38481497>.
>